### PR TITLE
convert more metrics

### PR DESCRIPTION
**Pull request by snopoke**
Name changes:

commcare.case_importer.case duration tag
  -> commcare.case_importer.duration_per_case duration tag

### CODE REVIEW

File: corehq/apps/couch_sql_migration/tests/test_migration.py [hunk 1]
```diff
@@ -781,17 +781,17 @@ def test_v1_case(self):
...
             self._do_migration_and_assert_flags(self.domain_name)
         tracked_stats = [
-            'commcare.couch_sql_migration.unprocessed_cases.count.duration:',
-            'commcare.couch_sql_migration.main_forms.count.duration:',
-            'commcare.couch_sql_migration.unprocessed_forms.count.duration:',
-            'commcare.couch_sql_migration.count.duration:',
+            'commcare.couch_sql_migration.unprocessed_cases.count',
+            'commcare.couch_sql_migration.main_forms.count',
+            'commcare.couch_sql_migration.unprocessed_forms.count',
+            'commcare.couch_sql_migration.count',
```
Comment (snopoke) [inline corehq/apps/couch_sql_migration/tests/test_migration.py]: @millerdev FYI this no longer checks for the duration tag, only the metric name. Let me know if you think that's an issue.

File: corehq/elastic.py [hunk 2]
```diff
@@ -9,6 +9,7 @@
 
 from memoized import memoized
 
+from corehq.util.datadog.gauges import datadog_counter
```
Comment (millerdev) [inline corehq/elastic.py]: curious why this one wasn't changed?
```suggestion
from corehq.util.metrics import metrics_counter
```
Will also need to change
https://github.com/dimagi/commcare-hq/blob/69fb53ba2ebf60056ad4bffd431a8aabd4d3531b/corehq/elastic.py#L508

File: corehq/util/metrics/tests/utils.py [hunk 3]
```diff
@@ -15,3 +17,12 @@ def get_tag_pairs(tags: list):
...
         yield stats
+
+
+@contextmanager
+def capture_metrics():
+    from corehq.util.metrics import _metrics
+    capture = DebugMetrics(capture=True)
+    _metrics.append(capture)
+    yield capture.metrics
+    _metrics.pop()
```
Comment (millerdev) [inline corehq/util/metrics/tests/utils.py]:
```suggestion
    try:
        yield capture.metrics
    finally:
        assert _metrics[-1] is capture, _metrics
        _metrics.pop()
```

**Review by millerdev — APPROVED** [review]
Approving on condition that `capture_metrics` suggestion is accepted.

File: corehq/elastic.py (hunk 2 above)
Comment (snopoke) [reply corehq/elastic.py]: I wanted to get these other changes out first

**Pull request merged**